live-examples/js-examples/intl/intl-locale.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">const korean = new Intl.Locale("ko", {script: "Kore", region: "KR", hourCycle: "h24", calendar: "gregory"});
+const japan = new Intl.Locale("ja-Jpan-JP-u-ca-japanese-hc-h12")
+console.log(korean.baseName)
```

Comment:
Also a blank line before this (after the 2 declarations) would help readability.